docs/blogs/EnhancedDevPreviewGatewayAPI/GettingCreative.md

```diff
@@ -0,0 +1,263 @@
+# Getting Creative
+
+In the previous section, we showed how to create a simple HTTPRoute. Let’s expand upon this example and include more
+advanced usages of Gateway API.
+
+## Route Matching
+Gateway API is a powerful API for directing traffic based on request characteristics such as path, headers, query
+parameters, and HTTP methods. Let’s explore how we can use these functions to direct traffic to different backends.
+
+#### Path Based Routing
+In this first example, we will match traffic based on the paths `/foo` and `/bar`. Path-based routing is useful for
+segmenting a website into various backend services. In the current OpenShift Route API, we must make a Route per path:
+```yaml
+apiVersion: v1
+kind: Route
+metadata:
+  name: http-foo
+  namespace: demo-app
+spec:
+  host: http.gwapi.${DOMAIN}
```

Comment:
This host should match the other route, correct?  And the OpenShift Route object shouldn't be using a domain with "gwapi" in it.  
```suggestion
  host: http.${DOMAIN}
```